doc/contributing/submitting-changes.chapter.md

```diff
@@ -82,6 +82,48 @@ If a security fix applies to both master and a stable release then, similar to r
 
 Critical security fixes may by-pass the staging branches and be delivered directly to release branches such as `master` and `release-*`.
 
+## Deprecating/removing packages {#submitting-changes-deprecating-packages}
+
+When software projects get discontinued or are unmaintained and break, we have to remove them. Another reason would be that there is no maintainer for the package in nixpkgs. In that case, try to find a new maintainer first.
```

Comment:
```suggestion
When software projects got discontinued, are unmaintained, broken for a long time or have severe security vulnerabilities, we have the option to remove them. Another reason would be that for a longer time there is no maintainer for the package in nixpkgs. In any case, it should always be attempted to find a new maintainer or patch smaller issues first.
```

Maybe we could add examples when a package can definitely be removed or not.

Can be safely removed:
- Package is broken since one or several years and did not receive meaningful commits (commits that are not treewide changes.).
- ...

Can not be removed:
- broken since last week due to an unrelated change.
- ...